src/com/strongjoshua/console/AbstractConsole.java

```diff
@@ -188,6 +189,11 @@ public AbstractConsole () {
 					log(msg, LogLevel.ERROR);
 					if (consoleTrace) {
 						log(e, LogLevel.ERROR);
+					} else if (e.getCause() instanceof InvocationTargetException) {
+						Throwable originalException = e.getCause().getCause();
+						if (originalException != null) {
```

Comment:
What if `originalException` is null? What should be the appropriate action? Simply ignore?